### ISSUE TITLE
Bug with Recurrence  in "New expense payed with Loan/payment plan"

### ISSUE THREAD

**Issue by crazycks** [Eqonomize/Eqonomize#62]
Hello,

i would like to advise you that i have got this problem after creating of "New expense payed with Loan/payment plan"..
after closing of the soft..when i re-open, a crual error appears, ts prevents me from keeping the saved planning/schedule....
So i have to cancel these below lines to re-open the soft!!  can you help me and perhaps amend it?
many thanks 

<schedule id="101" revisions="9">
		<transaction type="mulltiaccount" date="2018-10-08" id="98" revisions="9" description="Crédit Conso" category="56" timestamp="1538862298087000">
			<transaction type="expense" date="2018-10-08" id="99" revisions="9" timestamp="1538862298087000" cost="610.6200" from="97"/>
			<transaction type="expense" date="2018-10-08" id="100" revisions="9" timestamp="1538862298087000" cost="38.8700" from="2" payee="Caisse d'Epargne"/>
		</transaction>
		<recurrence type="monthly" startdate="2018-10-08" enddate="2019-03-01" frequency="1" day="1" weekendhandling="0"/>
	</schedule>

**Comment by crazycks** [comment]
<schedule id="101" revisions="9">
		<transaction type="mulltiaccount" date="2018-10-08" id="98" revisions="9" description="Crédit Conso" category="56" timestamp="1538862298087000">
			<transaction type="expense" date="2018-10-08" id="99" revisions="9" timestamp="1538862298087000" cost="610.6200" from="97"/>
			<transaction type="expense" date="2018-10-08" id="100" revisions="9" timestamp="1538862298087000" cost="38.8700" from="2" payee="Caisse d'Epargne"/>
		</transaction>
		<recurrence type="monthly" startdate="2018-10-08" enddate="2019-03-01" frequency="1" day="1" weekendhandling="0"/>
	</schedule>

**Comment by hanna-kn** [comment]
Fixed. It was a simple typo - "mulltiaccount" should be "multiaccount".

An updated AppImage for GNU/Linux is available at https://transfer.sh/wsZcZ/Eqonomize-git.6e0e2fb-x86_64.AppImage.

**Comment by hanna-kn** [comment]
Note that a recurring "expense paid with loan/payment plan" is a bit odd (I'm considering removing the recurrence tab from the dialog). Each recurrence the purchase will be duplicated and the new loan will increase.

**Comment by crazycks** [comment]
Dear,
the link doesn't work
 https://transfer.sh/wsZcZ/Eqonomize-git.6e0e2fb-x86_64.AppImage.
But  i modified directly in XML file.

but it's very interresting to check a loan plan payment.

**Comment by hanna-kn** [comment]
> the link doesn't work
> https://transfer.sh/wsZcZ/Eqonomize-git.6e0e2fb-x86_64.AppImage.

That's weird. The link works for me.

> But i modified directly in XML file.

The error will recur each time you save the file.

**Comment by crazycks** [comment]
yes it's strange for the link, but perhaps that it's cause i am for now on Windows session and not on Linux session.

When i modified "budget.eqz" directly with Open++,i saved and after i was be able to re-open software without any problem. But i suppose  that if i creat a new loan plan i ill the same error.

**Comment by crazycks** [comment]
and is it  possible for you to change too?  on your download link page: Windows binary

**Comment by hanna-kn** [comment]
> When i modified "budget.eqz" directly with Open++,i saved and after i was be able to re-open software without any problem. But i suppose that if i creat a new loan plan i ill the same error.

No. The error will recur when any change is saved (the whole file is rewritten).

> and is it possible for you to change too? on your download link page: Windows binary

The Windows binary is not updated between releases (the creation cannot be automated as with the GNU/Linux binary). I will release a new version soon within the next few weeks. Until then I suggest you avoid creating a recurring expense paid with loan (you can create a regular recurring expense with a loan as account, or if it really is a single purchase with a payment plan you're after, then you should create a recurring debt payment instead).

**Comment by crazycks** [comment]
ok many thanks for you help!
for now i'll do that and i am waiting for next your update.
perhaps you will create a link for  Automatic updates of stock quotes ! ishould be very very good!!

thansk again!
